schema/ingestion_config/migrations/issue_997.py

```diff
@@ -0,0 +1,185 @@
+"""
+This script migrates the ingest config files to a new schema based on the changes outlines in
+https://github.com/chanzuckerberg/cryoet-data-portal/issues/997
+"""
+import glob
+import itertools
+import json
+import traceback
+from typing import Union
+
+import numpy as np
+import yaml
+
+
+def rawtilts_to_collection_metadata(config: dict) -> None:
```

Comment:
Could this change be added to https://github.com/chanzuckerberg/cryoet-data-portal-backend/blob/b8a851458ceee2fd49e7bfa69ce41230ce00c3a9/ingestion_tools/scripts/transform_ingestion_configs.py ?

Reply:
the whole script or just this function?

Reply:
I believe this is the only method that hasn't been ported over to the script.